app/components/find/courses/entry_requirements_component/view.html.erb

```diff
@@ -3,32 +3,41 @@
   <h3 class="govuk-heading-m">Qualifications needed</h3>
   <div data-qa="course__required_qualifications">
 
-    <% if course.degree_grade.nil? && course.additional_degree_subject_requirements.nil? %>
+    <% unless course.teacher_degree_apprenticeship? %>
```

Comment:
This component is becoming unreadable and prone to bugs because of all the conditionals. I think there are actually two components here `tda_entry_level_requirements_component` and `post_graduate_entry_level_requirements`.

Reply:
I agree. Reading this component was painful. Too much ifs and elses and difficult to follow.

Adding my TDA change was more pragmatic because if I move the postgraduate to another component it might scope creep. 

I can however create the TDA entry level requirements, I think is a good idea. Wdyt?

Reply:
I think having a TDA entry level requirements is definitely better than what we have here, good idea.

Reply:
Done!